### PR TITLE
Some initial work for 1.99

### PR DESCRIPTION
**Pull request by matterhorn103**
Aimed for more systematic short internal names and tried to make sure they match the filenames, hope that's ok

### CODE REVIEW

File: source/docs/getting-started/making-selections.md [hunk 1]
```diff
@@ -1,21 +1,25 @@
...
 ![](../../_static/tutorial-select-tool.png)
 
-The select provides you with the ability to select a single atom within a molecule. This is achieved by left clicking the atom. Press and holding the "shift" key allows for the selection of multiple atoms.
+The Selection Tool provides you with the ability to select a single atom within a molecule. This is achieved by left-clicking the atom. Pressing and holding <kbd>shift</kbd> while clicking allows for the selection of multiple atoms.
 
-Right clicking on the black display will clear the selection made.
+Right-clicking on an empty part of the view will clear the selection made.
+
+```{warning}
+A known bug means that on some systems, multiple left-clicks are required to clear the selection.
```
Comment (ghutchis) [inline source/docs/getting-started/making-selections.md:12]: This should be fixed in 1.99, right?

**Comment by ghutchis** [comment]
I think this is a great start, thanks. One thing that's obvious is to add some sort of spelling check / grammar check as a GitHub action before approval.

File: source/docs/getting-started/making-selections.md (hunk 1 above)
Comment (matterhorn103) [reply source/docs/getting-started/making-selections.md:12]: Hmm, good point, I haven't actually explicitly checked, but equally hadn't noticed it suddenly working correctly either. I'll have a look on Monday.

Comment (matterhorn103) [reply source/docs/getting-started/making-selections.md:12]: I checked and it's not fixed, still behaves the same on my system.

**Pull request merged**